apps/system/style/window.css

```diff
@@ -337,9 +337,15 @@
   opacity: 1;
 }
 
+.appWindow.in-task-manager.overlay.no-screenshot > .identification-overlay {
+  visibility: visible;
+  opacity: 1;
```

Comment:
as opposed to the rest of the time where we make the overlay appear with no transition and only transition if we're removing it.